src/bot.py

```
@@ -6,784 +6,740 @@
 import time
 from base64 import b64encode
 from datetime import datetime
+from typing import Any, Dict, List, Literal, NotRequired, Optional, Sequence, Set, Tuple, TypedDict
```

Ruff (0.14.10)

Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)